source/numerics.tex

```diff
@@ -16736,6 +16741,43 @@
   template<@\exposconcept{math-floating-point}@ V>
     @\exposid{deduced-simd-t}@<V>
       sph_neumann(const rebind_simd_t<unsigned, @\exposid{deduced-simd-t}@<V>>& n, const V& x);
+
+  // \ref{simd.bit}, Bit manipulation
+  template<@\exposconcept{simd-type}@ V> constexpr V byteswap(const V& v) noexcept;
```

Comment:
Out of curiosity: Why this "simd-type" formulation instead of taking a suitably-parameterized `std::basic_simd` parameter type?

Reply:
I think I asked that question in LWG but I don't recall an answer. What difference does it make (other than style)?